release-content/0.14/migration-guides/11278_Fix_fit_canvas_to_parent.md

```diff
@@ -1 +1 @@
-Cancels the migration from https://github.com/bevyengine/bevy/pull/11057
+`Window::fit_canvas_to_parent` was removed in 0.13, but that ended up being problematic because many users depended on its behavior when they could not customize the CSS. It is now re-added to account for this need.
```

Comment:
Took me a bit to realize this is for WASM builds and not about bevy UI.

```suggestion
`Window::fit_canvas_to_parent` (expanding the canvas bevy is drawing to in a WASM/web build) was removed in 0.13. That was problematic because many users depended on its behavior when they could not customize the CSS. It is now re-added to account for this need.
```